src/app/community/_components/Comment.tsx

```diff
@@ -1,29 +1,114 @@
 import { calculateTimeDifference } from '@/libs/calculateDate';
+import { VerticalTripleDotIcon } from '@/public/index';
+import type { Users } from '@/types/userType';
+import { PopOver } from '@/components';
 
+import { deleteComment } from '@/api/communityAPI';
+import { toast } from 'react-toastify';
 import styles from './Comment.module.scss';
 
 const cn = classNames.bind(styles);
 
 interface CommentProps {
+  cardId: number;
+  userId: number;
+  commentId: number;
   nickname: string;
-  profile?: string;
+  profile: string | null;
   createdTime: string;
   comment: string;
 }
 
-export default function Comment({ nickname, profile, createdTime, comment }: CommentProps) {
+interface UserDataType {
+  data: Users;
+  status: string;
+  message: string;
+}
+
+export default function Comment({ cardId, userId, commentId, nickname, profile, createdTime, comment }: CommentProps) {
+  const queryClient = useQueryClient();
   const createdTimeToDate = new Date(createdTime);
+  const [isOpenPopOver, setIsOpenPopOver] = useState(false);
+
+  const userData = queryClient.getQueryData<UserDataType>(['userData']);
+
+  const userID = userData?.data.id;
+
+  const { mutate: deleteCommentMutation } = useMutation({
+    mutationFn: deleteComment,
+    onSuccess: () => {
+      toast.success('댓글이 삭제되었습니다.');
+      queryClient.invalidateQueries({
+        queryKey: ['postData', cardId],
+      });
+    },
+    onError: () => {
+      toast.error('댓글 삭제 중 오류가 발생하였습니다.');
+    },
+  });
+
+  const handleClickPopOver = (e: MouseEvent<HTMLDivElement>) => {
+    setIsOpenPopOver(!isOpenPopOver);
+    e.stopPropagation();
+  };
+
+  const handleClosePopOver = () => {
+    setIsOpenPopOver(false);
+  };
+
+  const handleClickDelete = (e: MouseEvent<HTMLDivElement>) => {
+    deleteCommentMutation(commentId);
+    e.stopPropagation();
+  };
+
+  const handleClickReport = (e: MouseEvent<HTMLDivElement>) => {
+    e.stopPropagation();
+  };
+
+  const MY_POPOVER_OPTION = [
+    {
+      label: '삭제하기',
+      onClick: handleClickDelete,
+    },
+    {
+      label: '신고하기',
+      onClick: handleClickReport,
+    },
+  ];
+
+  const OTHERS_POPOVER_OPTION = [
+    {
+      label: '신고하기',
+      onClick: handleClickReport,
+    },
+  ];
+
   const timeAgo = calculateTimeDifference(createdTimeToDate);
   return (
     <div className={cn('container')}>
       <ProfileImage profileImage={profile || null} />
       <div className={cn('content-wrapper')}>
-        <div className={cn('user-info')}>
-          <p className={cn('nickname')}>{nickname}</p>
-          <p className={cn('time-ago')}>{timeAgo}</p>
+        <div className={cn('user-info-wrapper')}>
+          <div className={cn('user-info')}>
+            <p className={cn('nickname')}>{nickname}</p>
+            <p className={cn('time-ago')}>{timeAgo}</p>
+          </div>
+          <div className={cn('dot-icon')} onClick={(e) => handleClickPopOver(e)}>
```

Comment:
사소하지만 특별히 추가되는 parameter가 없을 경우에는 그냥 넣어도 되는 걸로 알고 있습니다
```suggestion
          <div className={cn('dot-icon')} onClick={handleClickPopOver}>
```

Reply:
이벤트 버블링 막으려구 추가했슴당,,